db/seeds.rb

```diff
@@ -76,19 +76,21 @@
 # responsibility override workflow
 
 CaseInformation.find_or_create_by!(nomis_offender_id: 'G7658UL') do |info|
-    info.tier = 'A'
-    info.case_allocation = 'NPS'
-    info.welsh_offender = 'Yes'
-    info.manual_entry =  true
-    info.team_id = team1.id
+  info.tier = 'A'
+  info.case_allocation = 'NPS'
+  info.welsh_offender = 'Yes'
```

Comment:
Shouldn't `welsh_offender` be removed from these seeds, given that we've now added the `probation_service` field?